pact/col-template.pact

```diff
@@ -139,25 +139,14 @@
   )
 
   (defun has-remote-router:string (domain:integer)
-    (with-default-read routers (int-to-str 10 domain)
-      {
-        "remote-address": "empty"
-      }
+    (with-read routers (int-to-str 10 domain)
       {
         "remote-address" := remote-address
       }
-      (enforce (!= remote-address "empty") "Remote router is not available.")
       remote-address
     )
   )
 
-  ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; GasRouter ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-  (defun quote-gas-payment:decimal (domain:integer)
-    (has-remote-router domain)
-    (igp.quote-gas-payment domain)
-  )
-

```

Comment:
unused?

Reply:
I'm not sure i like the `quote-gas-payment` payment change since it seems like a signficant refactor. Is there a justification for the refactor aside from "we can make this tighter"? It introduces quite a bit of churn in this PR, and that's really the only issue i have with the code changes so far.